Bukkit/src/main/java/com/plotsquared/bukkit/schematic/StateWrapper.java

```diff
@@ -230,10 +236,34 @@ public boolean restoreTag(@NonNull Block block) {
                 }
                 return false;
             }
+            case "skull" -> {
+                if (state instanceof Skull skull) {
+                    CompoundTag skullOwner = ((CompoundTag) this.tag.getValue().get("SkullOwner"));
```

Comment:
```suggestion
                    CompoundTag skullOwner = ((CompoundTag) this.tag.getValue().get("SkullOwner"));
```
Do we still support 1.15.2? If so, this needs a special handling, the filed used to be called "Owner" in prior versions.

Reply:
1.15 can't run java 17 at all